specification/confidentialledger/resource-manager/readme.md

```diff
@@ -69,11 +69,14 @@ This is not used by Autorest itself.
 ```yaml $(swagger-to-sdk)
 swagger-to-sdk:
   - repo: azure-sdk-for-net
+  - repo: azure-sdk-for-net-track2
   - repo: azure-sdk-for-python-track2
   - repo: azure-sdk-for-java
+  - repo: azure-sdk-for-java-track2
```

Comment:
@weidongxu-microsoft does jave track2 use this way ?

Reply:
Musabbir is updated it and track2 removed.

This is correct for Java.